src/JsParsing/DictionaryProcessingJsExpressionParser.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\VueJsTemplating\JsParsing;
+
+use Peast\Peast;
+
+class DictionaryProcessingJsExpressionParser extends PeastJsExpressionParser implements JsExpressionParser {
+
+	/** @inheritDoc */
+	public function parse( $expression ) {
+		$pexp = Peast::ES2017( 'var testDict = ' . $expression )->parse();
```

Comment:
I’m not convinced the whole dispatching business is necessary… can’t we wrap the expression in `(…)` to force an expression (rather than statement) context, and then always take the expression out of that?

```php
$parsedExpression = Peast::ES2017("($expression)")->parse()->getBody()[0]->getExpression()->getExpression();
```